spigot/src/main/kotlin/com/mineinabyss/geary/minecraft/actions/DealDamageAction.kt

```diff
@@ -23,7 +23,9 @@ public data class DealDamageAction(
 ) : GearyAction() {
     override fun runOn(entity: GearyEntity): Boolean {
         val bukkit = entity.toBukkit<LivingEntity>() ?: return false
-        bukkit.health = (bukkit.health - damage.randomOrMin()).coerceAtLeast(minHealth)
+        val ignoreArmor = false
```

Comment:
A singled out boolean like this will always be false right :p Move it to the class parameters, so that it's can be specified in the config.

Reply:
big question, where're the class params?

Reply:
also do I need to add the bool to every config that uses deal_damage now or is that only to specify a value other than default?

Reply:
```
@Serializable
@SerialName("geary:deal_damage")
public data class DealDamageAction(
    val damage: @Serializable(with = DoubleRangeSerializer::class) DoubleRange,
    val minHealth: Double = 0.0,
) : GearyAction() {
```

`damage` and `minHealth` are configurable parameters here. And yeah, as long as you give the parameter a default value you don't need to add it to the configs. Setting the default value to `false` is fine.

Reply:
implemented

Reply:
👍